src/Utility/TextProcessingTrait.php

```diff
@@ -12,7 +12,7 @@ trait TextProcessingTrait
      * @param bool $ellipsis
      * @return bool|string
      */
-    public function createSummary($text, $size = 300, $ellipsis = false)
+    public function createSummary($text, $size = 300, $ellipsis = false, $withBreakPoints = false)
```

Comment:
Adding this new parameter `withBreakPoints` which is always false would be the same as just removing the full if block because it's never called?

Reply:
Yes it does 😅 .
Just wanted to keep a track of the code if it's necessary...

Reply:
I would rely on version control for a good paper trail 😉

Reply:
You're right ;)